app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/CoInsuredList.kt

```diff
@@ -32,11 +30,9 @@ internal fun CoInsuredList(
         onEdit = {},
       )
     }
-    Divider(Modifier.padding(horizontal = 16.dp))
-    uiState.coInsured.forEachIndexed { index, coInsured ->
-      if (index != 0) {
-        Divider()
-      }
+
+    uiState.coInsured.forEach { coInsured ->
+      Divider()
```

Comment:
Do we want a divider at the top even if there are no more coinsured rows above it?

Reply:
We can assume the member is not null here. The state could be modeled better by using a sealed class with less non-nullable fields though.